Client/src/Header.jsx

```diff
@@ -0,0 +1,208 @@
+import React from 'react';
+import styled from 'styled-components';
+import CheckMark from '../assets/images/checked.png';
+import Like from '../assets/images/like.png';
+
+const Wrapper = styled.section`
+width: 66.66666667%;
+float: left;
+position: relative;
+min-height: 1px;
+padding-left: 10px;
+padding-right: 10px;
+box-sizing: border-box;
+display: block;
+-webkit-font-smoothing: antialiased;
+font-family: "Calibre", Helvetica, Arial, sans-serif;
+font-size: 16px;
+line-height: 1.42;
+color: #333333;
+background-color: white;
+-webkit-tap-highlight-color: transparent;
+`;
+
+export const CampName = styled.h1`
+margin-top: 0;
+font-weight: 700;
+font-size: 4.8rem;
+margin-bottom: 11px;
+font-family: inherit;
+line-height: 1;
+color: inherit;
+margin: 0.67em 0;
+    margin-right: 0px;
+    margin-left: 0px;
+box-sizing: border-box;
+display: block;
+    margin-block-start: 0.67em;
+    margin-block-end: 0.67em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+-webkit-font-smoothing: antialiased;
+-webkit-tap-highlight-color: transparent;
+`;
+
+const RecommendedListing = styled.div`
+white-space: nowrap;
+overflow: hidden;
+display: flex;
+flex-wrap: wrap;
+justify-content: space-between;
+margin-top: -20px;
+`;
+const Left = styled.div`
+display: flex;
+align-items: center;
+margin-top: 20px;
+`;
+const PercentageDiv = styled.div`
+margin-right: 10px;
+border-bottom-right-radius: 2px;
+border-top-right-radius: 2px;
+vertical-align: middle;
+height: 38px;
+width: 65px;
+padding: 11px;
+padding-left: 7px;
+text-align: center;
+color: #fff;
+background-color: #40d9ac;
+border: 1px solid #3cc69e;
+border-bottom-left-radius: 2px;
+border-top-left-radius: 2px;
+`;
+
+const Span = styled.span`
+font-size: 0.9em;
+display: inline-block;
+font: normal normal normal 14px/1 FontAwesome;
+text-rendering: auto;
+-webkit-font-smoothing: antialiased;
+text-align: center;
+color: #fff;
+content: " (" attr(src) ")";
+`;
+const BasedOn = styled.div`
```

Comment:
I think that some of the names in your react styled components could be more descriptive and give a better idea of what they're styling, especially the divs that are named based on the sections of html they're styling. It's not super clear what BasedOn or PercentageDiv are styling.